src/docs/product/session-replay/getting-started.mdx

```diff
@@ -0,0 +1,28 @@
+---
+title: Set Up
+sidebar_order: 1
+redirect_from:
+  - /session-replay/setup/
+  - /session-replay/getting-started/
```

Comment:
Are these current links we want to preserve?
We usually use this when we're deleting a page, to keep the URL alive

Reply:
removed